packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -150,6 +150,10 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Table-cellContents {
+  display: flex;
+  justify-content: inherit;
```

Comment:
why inherit? i know it'll be stretch by default, but curious what above it is setting this. I'm not seeing it immediately

Reply:
`justify-content: inherit;` so it grabs the `justify-content` setting from its parent table cell wrapper. The parent table cell wrapper has the appropriate justify setting from the Column's `align`